### ISSUE TITLE
GoPhish URL in Word Docs, PDFs, Excel Docs etc.

### ISSUE THREAD

**Issue by W00glin** [gophish/gophish#1205]
First of all I just want to say thank you for such an incredible product. This is a truly excellent piece of software and I thoroughly enjoy using it.

I am not having an issue I just have a general inquiry about using GoPhish phishing links in word documents, excel spreadsheets pdfs etc. Essentially what I would like to do is take the phishing URL that is sent to users and then embed it into a document of some sort. I understand how to do this but what I am having issues with how to determine the correct URL to use. Unless I misunderstand and correct me if I am wrong but doesn't GoPhish generate a unique URL for each person the phishing email is being sent too? If that is the case is there a way for the GoPhish admins to determine that URL(s) and use them in other places? If it is not, could this be a consideration for future releases?

**Comment by snooze6** [comment]
I agree that it would be useful to know which URL is sent to each user or even create new URLs without actually sending mails in order to create new attacks like @W00glin is suggesting.

**Comment by jordan-wright** [comment]
Hey everyone!

I agree completely. Let me see what I can do. Right now, I figure I might have a checkbox when building a campaign to avoid sending emails. Then, I'll make sure that the results export will include the URL.

**Comment by W00glin** [comment]
> 
> 
> Hey everyone!
> 
> I agree completely. Let me see what I can do. Right now, I figure I might have a checkbox when building a campaign to avoid sending emails. Then, I'll make sure that the results export will include the URL.

That would be excellent. Keep up the great work @jordan-wright

**Comment by oded86w** [comment]
It's can be great to generate camping without send email .
And get link that not assign to email address .
And put the link in PDF/Word or maybe in desktop.ini , So we can put the files in DOK , And do a test of USB dropping .
